components/proxy/src/test/java/com/hotels/styx/routing/StaticPipelineBuilderTest.java

```diff
@@ -61,8 +63,7 @@ public void staticPipelineBuilderTest() {
 
     @Test
     public void buildsInterceptorPipelineForBackendServices() throws Exception {
-
-        HttpHandler handler = new StaticPipelineFactory(clientFactory, environment, registry, ImmutableList.of(), false).build();
+        HttpHandler handler = new StaticPipelineFactory(clientFactory, environment, registry, ImmutableList.of(), factory,false).build();
```

Comment:
```suggestion
        HttpHandler handler = new StaticPipelineFactory(clientFactory, environment, registry, ImmutableList.of(), factory, false).build();
```

Reply:
IntellJ argument name view tends to hide this kind of mistakes 🙄

Reply:
It's a very small thing, but it will probably be considered a checkstyle violation.